From 5a5f2b5be720e5048e4c4dedffd717d751319653 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Wed, 29 Nov 2017 14:10:16 +0800
Subject: [PATCH 2940/5242] MLK-17022 drm/panel: panel-simple: Correct JDI
 TX26D202VM0BWA panel display timing flags

commit  6543f3438aa2239aecb1c813c378ea48e71c440b from
https://source.codeaurora.org/external/imx/linux-imx.git

The JDI TX26D202VM0BWA panel works in data enable(DE) mode.
Apparently, the panel's data enable signal is active high
according to the panel spec.  This patch corrects the DE
signal polarity from active low to active high.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/panel/panel-simple.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/panel/panel-simple.c b/drivers/gpu/drm/panel/panel-simple.c
index abddd7b..bbc2f71 100644
--- a/drivers/gpu/drm/panel/panel-simple.c
+++ b/drivers/gpu/drm/panel/panel-simple.c
@@ -1274,7 +1274,7 @@ static void panel_simple_shutdown(struct device *dev)
 	.vfront_porch = { 3, 5, 10 },
 	.vback_porch = { 2, 5, 10 },
 	.vsync_len = { 5, 5, 5 },
-	.flags = DISPLAY_FLAGS_DE_LOW,
+	.flags = DISPLAY_FLAGS_DE_HIGH,
 };
 
 static const struct panel_desc jdi_tx26d202vm0bwa = {
-- 
1.7.9.5

